docs/source/quickstart.md

```diff
@@ -0,0 +1,37 @@
+(quickstart)=
+## Quickstart
+Robbie now uses Nextflow to manage the workflow and can be run on a local system or a supercomputing cluster. You can use a container via singularity, docker, or the host's software. The current development cycle tests Robbie using singularity on an HPC with the Slurm executor - other setups *should* work but haven't been extensively tested.
+
+### `images.txt`
+Before running Robbie, you will need to create a text file that contains the paths to each image to be processed. For example, if within the "Robbie" parent directory there exists a folder named "images" containing the `.fits` files:
+
+``` bash
+ls images/* > images.txt
+```
+
+will populate `images.txt` with the image paths relative to the parent directory.
+
+### `robbie.nf`
+This file describes the workflow and can be inspected but shouldn't be edited directly. To describe the command line arguments, use
+``` bash
+robbie.nf --help
+```
+
+### `nextflow.config`
+This file is the configuration setup and contains all the command line arguments' default values. You can change these defaults by copying the `nextflow.config` and editing the relevant params.\<argument\>. You can then use your custom config via:
+``` bash
+nextflow -C my.config run robbie.nf
+```
+The `-C my.config` directs Nextflow to use *only* the configuration described in `my.config`. If you use `-c`, then it will also read the `nextflow.config` file.
+
+### `-profile`
+
+If you're running Robbie on your local machine, you should use the `-profile local` option to use the Robbie docker image. For example:
+
+``` bash
+nextflow -C my.config run robbie.nf -profile local
+```
+
+If you're running Robbie on a supercomputing cluster (HPC), you should use the relevant cluster profile (`-profile zeus` or `-profile magnus`) to assure you're using the cluster's job queue (such as Slurm). If there isn't a profile for your cluster (check in `nextflow.config`), you may have to make your own.
+
+Additional configuration files are stored in the `./config` directory and may be useful templates for your work.
```

Comment:
Maybe we should rip all of this out now. Instead, make the -profile local the default, i.e. if the config does not recognise a supercomputer hostname, then assume you're using a local machine and use the local profile settings.

Reply:
Does Nextflow check for a hostname environment variable, for example `$HOSTNAME`? I'll look into making this the default behaviour.

Reply:
For usability I like Nicks idea. The only draw back is that we'll have to provide/update the environment settings for each cluster that people will want to use. Can we get around this by having either a separate config file that just defines the different cluster profiles, and/or a description in the documentation of how/where to update/add to the profiles for different clusters?

Reply:
I vote for showing how you would make a new cluster profile. Probably helps with portability so rather than us having to define cluster profiles people can define it themselves?

Reply:
This portion of code Nick wrote is quite useful though: 

```
if ( "\$HOSTNAME".startsWith("garrawarla") ) {
profile...
}
```

I guess it depends how often it is used outside Pawsey as realistically we could add those if statements for each Pawesy cluster.

Reply:
Yep, I've made different `$HOSTNAME` checks for each supercomputer in other pipelines I've made and ti works well. I think doing this all in one `nextflow.config` file is easiest but should be documented so it's not some magic behind the scenes that the user doesn't understand. I can write this part of the documentation if you prefer?

Reply:
Please feel free. I'll make the other changes, do the PR and you can make a new branch!